docker/manifests/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: drupal-bootstrap
+  labels:
+    app: bootstrap
+spec:
+  replicas: 1
+  selector: 
+    matchLabels:
+      app: bootstrap
+  template:
+    metadata:
+      labels:
+        app: bootstrap
+    spec:
+      volumes:
+        - name: app-code
+          emptyDir: {}
+        - name: localsettings
+          secret:
+            secretName: bootstrap-localsettings
+      containers:
+        - name: fpm
+          image: quay.io/kindlehl/php-fpm:bootstrap-v5
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 9000
+          volumeMounts:
+            - mountPath: /var/www/drupal-bootstrap.org
+              name: app-code
+        - name: app
+          image: quay.io/kindlehl/bootstrap:beta-0.0.34
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 80
+          volumeMounts:
+            - mountPath: /var/www/drupal-bootstrap.org
+              name: app-code
+      initContainers:
+        - name: app-init
+          image: quay.io/kindlehl/bootstrap:beta-0.0.34
+          imagePullPolicy: Always
+          volumeMounts:
+            - mountPath: /var/www/drupal-bootstrap.org
+              name: app-code
+            - mountPath: /localsettings
+              name: localsettings
+          command:
+            - /bin/sh
+            - -c
+            - cp -Ra /var/www/drupal-bootstrap.org-build/* /var/www/drupal-bootstrap.org && cp /localsettings/settings.local.php /var/www/drupal-bootstrap.org/docroot/sites/default
```

Comment:
This is not a good way to handle the local settings secret, as it will mean updates to the secret will not be active in the pod until a restart. We should just mount the secret at its appropriate location.

Reply:
I think this is a good way to do it. If we are going to pursue containerization, I think it would be best to embrace the idea of having an immutable infrastructure, and getting as close to an ideal '12 factor app' process as we can.

Reply:
I think you misunderstand. If you copy the secret from its mount location to another location, any update of that secret through kubectl will not be reflected. This is very confusing, doesn't match standards and is just not correct. If you want the secret to be readOnly on the mount, you can do that, and if you look at the secret and how its mounted in the helm chart, I am actually doing that.

Reply:
I did misunderstand. I thought you were suggesting storing localsettings on a mounted EFS volume.

Reply:
+1